docs/installation/helm.md

```diff
@@ -1,75 +1,157 @@
-# Helm 
+# Helm
 
+This comprehensive guide will help you install SigLens on your local Minikube Kubernetes cluster using Helm. Helm simplifies the deployment of Kubernetes applications, making it a breeze to set up SigLens.
 
+The Helm approach is advantageous for users managing Kubernetes clusters who want a simplified and standardized way to deploy SigLens. Helm simplifies SigLens deployment as a Kubernetes package, providing standardized, scalable, and version-controlled installations. This method is best suited for users who leverage Kubernetes orchestration, offering templating and packaging for easy configuration and sharing within a Kubernetes-native environment.
 
+### Prerequisites
 
+Before installing SigLens, ensure you have the following tools installed on your system:
 
-```
-$ helm repo add siglens-repo https://siglens.github.io/charts
-```
-
-
-```
-$ helm repo update
-```
-If you want to install using <strong>Minikube</strong>, follow the steps below:
-#### Prerequisites
-Before we start the installation, make sure you have the following tools installed on your system:
-- [Kubernetes command-line interface (CLI)](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Kubernetes command-line interface (kubectl)](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
```

Comment:
Not required, as kubectl comes bundled with minikube. Still, it's better to mention this explicitly

Reply:
Or move this below docker, and mention it as a optional step, since kubectl is bundled with minikube/kind